src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -240,5 +240,53 @@ <H2>Address Cache</H2>
 <P>Since these 2 properties are part of the security policy, they are
 not set by either the -D option or the {@code System.setProperty()} API,
 instead they are set as security properties.</P>
+<a id="Unixdomain"></a>
+<H2>Unix domain sockets</H2>
+<P>There are a number of system (and networking) properties that affect the behavior of
+channels to <i>Unix domain</i> server sockets when binding <i>automatically</i>.
+<p>
+Automatic binding of a server socket occurs when {@link 
+java.nio.channels.ServerSocketChannel#bind(SocketAddress,int) ServerSocketChannel.bind}
+is called with a {@code null} address parameter. In this case, the system
```

Comment:
doesn't it also happen when it is called with a UnixDomainSocketAddress that has an empty path?

Reply:
No, an empty path signifies an unnamed address and only client sockets (SocketChannels) are allowed to be unnamed.

Reply:
You get a BindException if you try to bind a ServerSocketChannel to the unnamed address. But, the exception message is "unhelpful". I would like to fix that.

Reply:
Ah good.